docs/installation/getting-started/quick-start.md

```diff
@@ -4,318 +4,1720 @@
 </info-box>
 
-### Download the customized build
+### The CKEditor 5 superbuild limitations
 
-This is as simple as it gets: just press the **Start** button to download your customized package.
+CKEditor 5 provides a multitude of plugins offering {@link features/index various features} addressing versatile needs. While the superbuild is designed to provide as many of them as possible, some of these plugins may conflict with each other. Due to that fact, several of those needed to be excluded from the superbuild and are not available that way.
 
-Refer to the *What's next?* note at the end of this page for further steps in your CKEditor 5 journey!
+The plugins not available currently in the superbuild include:
+* Watchdog
+* ContextWatchdog
+* Context
+* Title
 
-## Building the editor from source
+### Using the CKEditor 5 superbuild
 
-This scenario allows you to fully control the building process of CKEditor 5. This means that you will not actually use the builds introduced in the previous path, but instead build CKEditor from source directly into your project. This integration method gives you full control over which features will be included and how webpack will be configured.
+To start using the CKEditor 5 superbuild from CDN, first add the editor placeholder to your document.
 
-This is an advanced path that assumes that you are familiar with npm and your project uses npm already. If not, see the [npm documentation](https://docs.npmjs.com/getting-started/what-is-npm) or call `npm init` in an empty directory and check the result.
+```html
+<div id="editor"></div>
+```
 
-### Setting up the environment
+Then, include the code from CDN to superbuild to load the editor.
 
-Before moving to the integration, you need to prepare three files that will be filled with code presented in this guide. Create the `webpack.config.js`, `app.js` and `index.html` files.
+```html
+<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/super-build/ckeditor.js"></script>
+```
 
-Then, install packages needed to build CKEditor 5:
+In the superbuild, all editor classes are stored under the `CKEDITOR` object. To create the classic editor, you need to access the `CKEDITOR` object first, then call the {@link module:editor-classic/classiceditor~ClassicEditor#create `ClassicEditor.create()`} method.
 
-```bash
-npm install --save \
-	css-loader@5 \
-	postcss-loader@4 \
-	raw-loader@4 \
-	style-loader@2 \
-	webpack@5 \
-	webpack-cli@4
+```html
+<script>
+	CKEDITOR.ClassicEditor
+		.create( document.querySelector( '#editor' ) )
+		.catch( error => {
+			console.error( error );
+		} );
+</script>
```

Comment:
This section suggests that doing those steps is enough to make the editor work (which is true for the previous tutorial about using CDN, it is possible to just paste similar lines to an HTML document and it is working), for this guide it is not true and misleading. When you paste just those steps you start receiving errors about missing config of multiple plugins. e.g. RH. I see two options here: delete those steps and refer to full examples or make a disclaimer that due to the complex character of the superbuild there are more configurations to do which are explained in detail below. At this stage, it might be very confusing for readers who do not have experience with editors and want to give it a try. 

I would go with the attitude to include a full fully configured example of an editor without premium features and then an example with premium features. We should focus on quick experience first and then go deep into details.